### ISSUE TITLE
[Feature request] Option to save fanart.jpg only

### ISSUE THREAD

**Issue by jimbb** [DoctorD1501/JAVMovieScraper#169]
For now, the scraper only have the option to save both fanart and poster, but it will be grateful if they can be separated into 2 options, fanart.jpg only and poster.jpg only as most of the time the poster is the cropped right-hand side of the fanart picture.

**Comment by throwaway987321** [comment]
I agree, and think you should have checkboxes for all three files (the .nfo file, too).
